.editorconfig

```diff
@@ -236,32 +236,35 @@ dotnet_naming_style.first_upper.capitalization = first_word_upper
 # prefix_interface_interface_with_i - Interfaces must be PascalCase and the first character of an interface must be an 'I'
 dotnet_naming_style.prefix_interface_interface_with_i.capitalization = pascal_case
 dotnet_naming_style.prefix_interface_interface_with_i.required_prefix = I
+# non_public_fields - Non-Public fields should be camel case and prefixed with m_
+dotnet_naming_style.non_public_fields.capitalization = camel_case
```

Comment:
Thanks for fixing these, I think this was the first editorconfig I tried to set up for c# and it had my head spinning :)